docker-compose.dev.yml

```
@@ -0,0 +1,35 @@
+services:
+  dev:
+    build:
+      dockerfile: Dockerfile
+      context: .
+      target: development
+    image: todo-app:dev
+    ports: 
+     - 3001:3001
```

Minor, but I think your app runs at 5000 by default?
```suggestion
     - 3001:5000
```

It would be nice to quickly mention in your Readme which port it should be running on locally for each example too